demos/interactive_face_detection_demo/cpp/main.cpp

```diff
@@ -81,12 +81,7 @@ int main(int argc, char *argv[]) {
                 visualizer.enableEmotionBar(emotionsDetector.emotionsVec);
         }
 
-        cv::VideoWriter videoWriter;
-        if (!FLAGS_o.empty() && !videoWriter.open(FLAGS_o, cv::VideoWriter::fourcc('M', 'J', 'P', 'G'),
-                                                  !FLAGS_no_show && FLAGS_fps > 0.0 ? FLAGS_fps : cap->fps(),
-                                                  frame.size())) {
-            throw std::runtime_error("Can't open video writer");
-        }
+        LazyVideoWriter videoWriter{FLAGS_o, cap->fps(), FLAGS_limit};
```

Comment:
In the demo the following expression was used for fps: 
`!FLAGS_no_show && FLAGS_fps > 0.0 ? FLAGS_fps : cap->fps()`

Reply:
Added